lib/global.js

```diff
@@ -1,14 +1,3 @@
-var createTorrent = require('create-torrent')
-var wrtc = require('wrtc')
+var wrtc = require('electron-webrtc')()
 
 global.WRTC = wrtc
-
-global.WEBTORRENT_ANNOUNCE = createTorrent.announceList
```

Comment:
No. This is there to add the websockets default trackers of createTorrent to any torrents. 
Without it, it will only use the trackers of the (existing) torrents, which mean for most torrents it won't announce to any websocket trackers, and won't be available from browsers.

(see https://github.com/feross/webtorrent-app/blob/c176e3897ac70d55acd099a4f15464fb85e79ed7/renderer/index.js#L34 )